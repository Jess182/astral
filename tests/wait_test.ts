import { launch } from "../mod.ts";

Deno.test("Wait for selector", async () => {
  // Launch browser
  const browser = await launch();

  // Open the webpage
  const page = await browser.newPage("http://deno.land");

  // Wait for selector
  const selected = await page.waitForSelector(".font-bold");
  console.log(selected);

  // Close browser
  await browser.close();
});
