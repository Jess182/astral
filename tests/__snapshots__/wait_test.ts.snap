export const snapshot = {};

snapshot[`Wait for function 1`] = `undefined`;
